create-cluster.rb

```diff
@@ -58,6 +58,9 @@ def install_components(cluster_name)
   dir = "terraform/cloud-platform-components"
   execute "cd #{dir}; rm -rf .terraform"
   switch_terraform_workspace(dir, cluster_name)
+  
+  # Comment disable_alerts() to enable high priority - Pagerduty alarms and lower prority Slack alerts for your cluster
```

Comment:
I'd move this comment from here, and down in the method body, add a comment above each 'sed' invocation, explaining its effect.

Also, will these leave your working copy in a state where there are outstanding changes?

Reply:
Yes, this will change the current working copy with an outstanding change. We have to ignore the  tfvars changes when pushing to git or checkout again after building the test cluster.